src/screens/onboarding/OnboardingScreen.tsx

```
@@ -62,6 +62,11 @@ export default function OnboardingScreen() {
         title={getString('onboardingScreen.complete')}
         mode="contained"
         onPress={() => {
+          console.log(
```

ESLint

[error] Unexpected console statement.

(no-console)